documentation/description.md

```diff
@@ -1225,6 +1227,57 @@ disruption ongoing. We can tune this by updating the `spec.maxPodUnavailable` pa
+cassandra     at org.apache.cassandra.service.StorageService.initServer(StorageService.java:632)
+cassandra     at org.apache.cassandra.service.CassandraDaemon.setup(CassandraDaemon.java:388)
+cassandra     at org.apache.cassandra.service.CassandraDaemon.activate(CassandraDaemon.java:620)
+cassandra     at org.apache.cassandra.service.CassandraDaemon.main(CassandraDaemon.java:732)
+cassandra ERROR [main] 2020-02-21 08:29:44,398 CassandraDaemon.java:749 - Exception encountered during startup
+cassandra java.lang.RuntimeException: A node with address /10.100.150.35 already exists, cancelling join. Use cassandra.replace_address if you want to replace this node.
+cassandra     at org.apache.cassandra.service.StorageService.checkForEndpointCollision(StorageService.java:577) ~[apache-cassandra-3.11.4.jar:3.11.4]
+cassandra     at org.apache.cassandra.service.StorageService.prepareToJoin(StorageService.java:823) ~[apache-cassandra-3.11.4.jar:3.11.4]
+cassandra     at org.apache.cassandra.service.StorageService.initServer(StorageService.java:683) ~[apache-cassandra-3.11.4.jar:3.11.4]
+cassandra     at org.apache.cassandra.service.StorageService.initServer(StorageService.java:632) ~[apache-cassandra-3.11.4.jar:3.11.4]
+cassandra     at org.apache.cassandra.service.CassandraDaemon.setup(CassandraDaemon.java:388) [apache-cassandra-3.11.4.jar:3.11.4]
+cassandra     at org.apache.cassandra.service.CassandraDaemon.activate(CassandraDaemon.java:620) [apache-cassandra-3.11.4.jar:3.11.4]
+cassandra     at org.apache.cassandra.service.CassandraDaemon.main(CassandraDaemon.java:732) [apache-cassandra-3.11.4.jar:3.11.4]
+```
+
+Following the [issue #170](https://github.com/Orange-OpenSource/casskop/issues/170), at least using Kubernetes and [Project Calico](https://docs.projectcalico.org/v3.9/getting-started/kubernetes/), we may fall into this issue, 
+for example using a fixed [ip pool](https://docs.projectcalico.org/v3.9/reference/resources/ippool) size.
+
+To manage this case we introduced the `restartCountBeforePodDeletion` CassandraCluster spec field which takes an `int32` as value.
+> Note : If you set it with a value lower or equals to 0, or if you omit it, no action will be performed
+
+In setting this field, the cassandra operator will check for each `CassandraCluster` if a pod is in a restart situation (based on restart count of the cassandra container inside the pod).
+In the case where the restartCount of the pod is greater than the value of `restartCountBeforePodDeletion` field and if we are in a Ip cross situation, we will delete the pod, which will be recreated by the Statefulset. 
+In the case of Project Calico usage, this allow to force the pod to get another available IP, which fix our bug. 
+
+### Ip Cross situation detection
+
+To detect that we are in a Ip cross situation, we add a new status field `CassandraNodeStatus` which will maintain a cache about the map of *Ip node* and his *hostId*, 
+based on Jolokia response, for all ready pods.
```

Comment:
I don't think telling that we use Jolokia to get this information is useful here. We can just remove it.